src/code_80040E50.c

```diff
@@ -1097,6 +1097,9 @@ void func_80042E00(s32 arg0, s32 arg1) {
 #else
 GLOBAL_ASM("asm/non_matchings/code_80040E50/func_80042E00.s")
 #endif
+   
+// UNUSED
+GLOBAL_ASM("asm/non_matchings/code_80040E50/func_80042EB8.s")
```

Comment:
Should we be generating the mips_to_c for unused functions?

Reply:
I usually do, if only because some day far in the future even unused functions will have to be decomped in order to reach 100%